Documentation/ApiOverview/Events/Events/Backend/_ModifyLinkExplanationEvent/_MyEventListener.php

```diff
@@ -15,7 +15,7 @@
 final readonly class MyEventListener
 {
     public function __construct(
-        private readonly IconFactory $iconFactory,
+        private IconFactory $iconFactory,
```

Comment:
Just searched in core. They also use the "readonly" keyword

```suggestion
        private readonly IconFactory $iconFactory,
```